### DISCUSSION TITLE
Backup

### DISCUSSION THREAD

**Discussion by Mano-Kk** [iSoron/uhabits#1765] [Feature Requests]
Is it possible to add backup functionality on a cloud. I want to temporarily delete the app, maybe a couple months but i will maybe need data when i come backup, so backup feature please

**Comment by adrienbeau** [comment]
Due to the extremely sensitive data managed by the app, and the Free Software ideals that permeate it, backup to a proprietary cloud (however popular) is unlikely to happen. See #104 for more discussion.

In your case, you can export your data from the app, and then import it later. And you can do whatever you want with your exported file, including putting it in a proprietary cloud.
